### DISCUSSION TITLE
How do I remove blue border from dropdown

### DISCUSSION THREAD

**Discussion by face0612** [heroui-inc/heroui#2573] [Q&A]
<img width="226" alt="Screenshot 2024-03-23 at 9 35 00 PM" src="https://github.com/nextui-org/nextui/assets/6833051/4d584745-ac17-4676-a46b-c4199b40516b">

How can I remove the blue border when hovering DropdownItem? Adding `border-none` to DropdownMenu or DropdownItem didn't work. This also sometimes appears on DropdownTrigger 

I did not add `color` or `variant` to DropdownMenu

**Comment by alphaxek** [comment]
Hi @face0612, can you please share the `Button variant` prop you are passing or code snippet, and NextUI version you are using.

**Reply by face0612** [reply]
I am using this structure

```
<Dropdown>
  <DropdownTrigger>
    <Image ... />
  </DropdownTrigger>
  <DropdownMenu className="bg-white rounded-md w-[200px] shadow absolute top-0 -right-5">
    <DropdownItem href="/settings" key="settings" className="p-2 text-base font-medium">
       settings
    </DropdownItem>
  </DropdownMenu>
</Dropdown>     
 ```
using version 2.2.9

**Comment by face0612** [comment]
this was resolved by adding focus-visible:outline-none

**Reply by ParvinEyvazov** [reply]
Thank you!
